### ISSUE TITLE
[CoE Starter Kit - BUG] Power App not showing up in CoE Toolkit

### ISSUE THREAD

**Issue by sawright02** [microsoft/coe-starter-kit#8785]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Power Apps needs ownership transferred to another resource. I can see the app in the environment under "shared with me". However, the app is not being shown in the CoE Toolkit. Please advise.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

1.4.8911-2406.3

### What app or flow are you having the issue with?

Manage Permissions

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Sorry I'm not following you. Are you saying you are transferring ownership of the kit's solution/apps/etc to another user? 
Or some unrelated app?

**Comment by sawright02** [comment]
My goal is to utilize the toolkit's Manage Permissions app to change the ownership of a single Power App. However, the app in question is not showing up in the inventory list.

**Comment by Jenefer-Monroe** [comment]
Gotcha thank you. It sounds like your inventory is incomplete. This could be due to a few issues so let's go through them.

**_Power Platform Admin Role_**

The kit only gathers information about the envts to which you have admin access. The Power Platform Admin Role, permanently and directly assigned, is required to fetch information about the tenant as a whole. Please see [What identity should I install the CoE Starter Kit with?](https://docs.microsoft.com/power-platform/guidance/coe/setup#what-identity-should-i-install-the-coe-starter-kit-with)

Can you please confirm that the user identity installing and running the flow has the Power Platform Admin Role assigned both directly and permanently? 
[Microsoft Azure Users](https://portal.azure.com/#view/Microsoft_AAD_UsersAndTenants/UserManagementMenuBlade/~/AllUsers) > Your User > Assigned Roles
Ensure Power Platform Administrator is Direct and Permanent
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/66096cf4-67ad-41c9-9900-0b3e46de0cee)

**Comment by sawright02** [comment]
Membership is set to group. End time is permanent.

**Comment by Jenefer-Monroe** [comment]
You will need to have the group be direct going forward as thats required for escalation of privileges. 
Please read this issue for more information: https://github.com/microsoft/coe-starter-kit/issues/8119

For this issue, do you see other apps from the environment? Is this the only app missing?

**Comment by sawright02** [comment]
Yes I see other apps, but not the one that I'm looking for. It appears as though no new apps in that specific environment are being picked up by the toolkit.

**Comment by Jenefer-Monroe** [comment]
what type of app is it? Canvas or MDA?

**Comment by sawright02** [comment]
Canvas

**Comment by Jenefer-Monroe** [comment]
Can you please go validate that these flows are running and passing? Look for failures.
Admin | Sync Template v4 (Driver)
Admin | Sync Template v4 (Apps)
SYNC HELPER - Apps

Note the GUIDs for the app, its envt and app guids
Then run this flow with them - SYNC HELPER - Apps
![image](https://github.com/user-attachments/assets/432ce98e-dff3-4f5d-ac38-08276451d93b)

**Comment by Jenefer-Monroe** [comment]
Closing out as no further action for starter kit team
